### DISCUSSION TITLE
GCN，GraphSage等的例子，训练结束后，怎么做fit；以及如何查看某个结点的分类、回归结果？

### DISCUSSION THREAD

**Discussion by jingmouren** [alibaba/GraphScope#133] [Q&A] [answered]
如题

**Comment by acezen** [comment, marked as answer]
when training finished, you can use the classifier of  machine learning library (e.g scikit-learn) to do the fit or get the classification/regression result.

example:   https://github.com/alibaba/graph-learn/blob/4906d022c18b0c2819363180b374b7e368ae5560/examples/eval/ppi_eval.py#L34-L46

**Reply by jingmouren** [reply]
> when training finished, you can use the classifier of machine learning library (e.g scikit-learn) to do the fit or get the classification/regression result.
> 
> 
如果中间过程的特征向量已经存到图结点，或者存到结构化表，是可以用后续的分类器。现在的例子模拟的是大图，用iterator逐批算了部分结果，最后再汇总得到准确率。得到的模型怎么再怎么做推断没看出来啊。

**Reply by baoleai** [reply]
Just like a general tf model, you can modify train.py, save ckpt during training, and then load ckpt during prediction. An example should be added here.
